tests/test_twitterCleanup.py

```diff
@@ -0,0 +1,26 @@
+from unittest import TestCase, mock
```

Comment:
`test_twitterCleanup.py` does not follow [PEP-8's module naming style](https://www.python.org/dev/peps/pep-0008/#package-and-module-names). Can we rename it as `test_twitter_cleanup.py`?